components/ws-manager-mk2/controllers/metrics.go

```diff
@@ -186,6 +185,7 @@ func (m *controllerMetrics) countWorkspaceStop(log *logr.Logger, ws *workspacev1
 	counter, err := m.totalStopsCounterVec.GetMetricWithLabelValues(reason, tpe, class)
 	if err != nil {
 		log.Error(err, "could not count workspace stop", "reason", "unknown", "type", tpe, "class", class)
+		return
 	}
 
 	counter.Inc()
```

Comment:
Would make sense to extend the metric with `status/outcome = OK/Error` such that we also capture when we've failed to actually capture this metric?